docs/guide/personalization.md

```diff
@@ -265,294 +264,285 @@ For the `content` interface one Content item is returned, for the `contenttypes`
-
-``` html+twig
-{% javascripts
-    '%kernel.root_dir%/../vendor/components/handlebars.js/handlebars.min.js'
-    '@EzSystemsRecommendationBundle/Resources/public/js/recommendationtemplaterenderer.js'
-    '@EzSystemsRecommendationBundle/Resources/public/js/recommendationtemplatehelper.js'
-    '@EzSystemsRecommendationBundle/Resources/public/js/recommendationrestclient.js'
-%}
-```
+    If the export is NOT successful, the Recommendation engine will not have the full content information.
+    This will break the recommendations. Even if the recommendations are displayed,
+    there is a big chance they won't have images, titles or deeplinks.
 
-Place a dedicated Twig helper in the place where you want to display recommendations:
+In order to display recommendations on your site, use a dedicated `showRecommendationsAction` from the `RecommendationController.php`:
 
 ``` html+twig
-{{ yc_show_recommendations(
-    contentId = content.id,
-    scenario = '',
-    limit = '',
-    contentType = '',
-    template = '',
-    fields = []
-) }}
+render_esi(controller(‘EzRecommendationClientBundle:Recommendation:showRecommendations’, {
+               ‘contextItems’: content.id,
+               ‘scenario’: ‘front’,
+               ‘outputTypeId’: ‘blog_post’,
+               ‘limit’: 3,
+               ‘template’: ‘EzRecommendationClientBundle::recommendations.html.twig’,
+               ‘attributes’: [‘title’, ‘intro’, ‘image’, ‘uri’]
+           }))
 ```
 
 ### Parameters
 
-| Parameter   | Type   | Description   |
-|-------------|--------|---------------|
-| `contentId`   | int    | In content-based views the Twig variable holding the content ID (the content you want to get recommendations for). |
-| `scenario`    | string | Scenario used to display recommendations. You can create custom scenarios in the [Personalization Engine's  dashboard](https://admin.yoochoose.net). |
-| `limit`       | int    | Number of recommendations to fetch. |
-| `contentType` | string | Content Type you are expecting in response, e.g. article. |
-| `template`    | string | Handlebars template name. Your templates are stored in the `ezsystems/recommendation-bundle/Resources/public/views` directory. Take a look at `default.html.twig` which includes a default template that can be used to prepare customized versions. |
-| `fields`      | array  | Fields which are required and will be requested from the recommender engine. These Field names are also used inside Handlebars templates. |
-
-Sample integration can take the following form:
-
-``` html+twig
-{{ yc_show_recommendations(
-    contentId = content.id,
-    scenario = 'popular',
-    limit = 5,
-    contentType = 'article',
-    template = 'default',
-    fields = ['uri', 'title', 'image', 'author', 'intro']
-) }}
-```
+| Parameter        | Type   | Description   |
+|------------------|--------|---------------|
+| `contextItems`   | int    | ID of the content you want to get recommendations for. |
```

Comment:
I don't like this parameter name. It should be contextItem because it represent only one value. I will change it in code and please change it also in doc.